server/helpers/utilities.js

```diff
@@ -29,7 +29,7 @@ function getAuthErrorDescription(error) {
     }`,
   )
   let type =
-    'Service temporarily unavailable. Please try again later. If this issue persists, please contact the Helpdesk on 0203 788 4636.'
```

Comment:
Guess you could potentially look at centralising the phone number and put it into config?